src/components/legal-requirements/legal-requirements.html

```diff
@@ -0,0 +1,12 @@
+<ion-row *ngFor="let req of requirements" align-items-center no-padding (swipe)="openCustomerHammgerConfig()">
```

Comment:
legal-requirements.ts doesn't have a openCustomerHammgerConfig() method

Reply:
good catch, I just copy and pasted code from dl25-prototype branch